src/internal/connector/uploadsession/uploadsession.go

```diff
@@ -57,15 +57,15 @@ func (iw *writer) Write(p []byte) (n int, err error) {
 		}).
 		SetBody(bytes.NewReader(p)).Put(iw.url)
 	if err != nil {
-		return 0, errors.Wrapf(err,
-			"failed to upload item %s. Upload failed at Size:%d, Offset: %d, TotalSize: %d ",
-			iw.id, rangeLength, iw.lastWrittenOffset, iw.contentLength)
+		return 0, clues.Wrap(err, "uploading item").WithAll(
+			"upload_id", iw.id,
+			"upload_chunk_size", rangeLength,
+			"upload_offset", iw.lastWrittenOffset,
+			"upload_size", iw.contentLength)
 	}
 
 	// Update last offset
 	iw.lastWrittenOffset = endOffset
 
-	logger.Ctx(context.Background()).Debugf("Response: %s", resp.String())
```

Comment:
Are we sure that we don't want to log this any longer?

Reply:
We can always put it back in if we need it.